deploy/sre-authorization/01-osd-sre-admin.ClusterRole.yaml

```diff
@@ -3,12 +3,15 @@ kind: ClusterRole
 metadata:
   name: osd-sre-admin
 rules:
+# SRE can get, list, and watch all resources
 - apiGroups:
   - ""
   resources:
   - '*'
   verbs:
-  - '*'
+  - get
```

Comment:
This (and the following 2 lines) appears to duplicate lines 20-22?

Reply:
Value for `apiGroup` is "" on line #8 and "*" on line 16.  They are not duplicates, though they could be merged.

Reply:
What I mean is does `*` cover `""`?

Reply:
Turns out `"*"` does cover `""`.  Removed `""`.